cts/cts-scheduler.in

```diff
@@ -389,6 +403,53 @@ function test_results {
     fi
 }
 
+
+if test -z "$single_test" && test "$num_jobs" -ne 1 \
+        && command -v parallel >/dev/null 2>&1; then
+    _ACC_FILE=()
+    _ACC_DESC=()
+    _ACC_ARGS=()
+    export -f do_test_inner error failed info normalize show_test
+    export base create_mode diff_opts failed io_dir test_cmd
+    export CRM_EX_OK CRM_EX_ERROR CRM_EX_NOT_INSTALLED \
+           CRM_EX_USAGE CRM_EX_NOINPUT
+    do_test() {
+        test "$num_jobs" -ne 0 || num_jobs=100%
+        test $# -gt 1 || test $# -lt 1 || test "$1" != "EOT" || {
+            parallel -j "$num_jobs" --halt soon,fail=100 --plus \
+              --tagstring '{= $_="\033[30;3".(++$::color%8)."m" =}' \
+              do_test_inner '{1}' \
+                            '{=2 $_="[".seq()."/".total_jobs()."] ".$_ =}' \
+                            '{3}' \
+                ::: "${_ACC_FILE[@]}" \
+                :::+ "${_ACC_DESC[@]}" \
+                :::+ "${_ACC_ARGS[@]}"
+            num_failed=$?
+            if test $num_failed -gt 100; then
+                error "More than 100 tests failed"
+            fi
+            return $num_failed
+        }
+        num_tests=$(( $num_tests + 1 ))
+        _ACC_FILE+=( "$1" )
+        _ACC_DESC+=( "$2" )
+        if test $# -gt 2; then
+            shift 2
+            _ACC_ARGS+=( "$*" )
+        else
+            _ACC_ARGS+=( "" )
+        fi
+    }
+    do_sep() { :; }
+else
+    test "$num_jobs" -le 1 \
+      || info "Need to serialize jobs despite %d jobs requested" "$num_jobs"
+    # allow alias expansion to prevent a need for trivial (inefficient) wrapper
+    shopt -s expand_aliases
+    alias do_test=do_test_inner
+    alias do_sep=echo
```

Comment:
Thinking about this more, the separators aren't used very consistently and don't really help anyway. Let's just ditch them.

Reply:
Also an option

Reply:
OTOH, in the serial output, they make sort of an anchor points when
skimming, I've always thought of that as feature, dunno.

Reply:
Keeping this tentative.

Reply:
Would miss those eye anchors when having a look at CI occassionally.
That would be a loosefull change.